packages/zoe/zoe.js

```diff
@@ -9,31 +9,28 @@ import { isOfferSafeForAll } from './isOfferSafe';
 import { areRightsConserved } from './areRightsConserved';
 import { evalContractCode } from './evalContractCode';
 import { makeTables } from './state';
-import { makeSeatMint } from './seatMint';
-import { makeEscrowReceiptConfig } from './escrowReceiptConfig';
+import { makeInviteConfig } from './inviteConfig';
 
 /**
  * Create an instance of Zoe.
  *
  * @param additionalEndowments pure or pure-ish endowments to add to evaluator
  */
 const makeZoe = (additionalEndowments = {}) => {
-  // Zoe has two mints: a mint for invites and a mint for
-  // escrowReceipts. The invite mint can be used by a smart contract
-  // to create invites to take certain actions in the smart contract.
-  // An escrowReceipt is an ERTP payment that is proof of
-  // escrowing assets with Zoe.
-  const {
-    seatMint: inviteMint,
-    seatAssay: inviteAssay,
-    addUseObj: addInviteSeat,
-  } = makeSeatMint('zoeInvite');
-
-  const escrowReceiptMint = makeMint(
-    'zoeEscrowReceipts',
-    makeEscrowReceiptConfig,
-  );
-  const escrowReceiptAssay = escrowReceiptMint.getAssay();
+  const makeInviteMint = () => {
+    const handleToSeat = new WeakMap();
```

Comment:
Why a `WeakMap` rather than a `PrivateName`?

Reply:
No particular reason. This code was ported from `seatMint.js` line 12, not written recently. I'll change it to a `PrivateName`.